src/audio/wav64.c

```diff
@@ -283,8 +294,10 @@ void wav64_open(wav64_t *wav, const char *fn) {
 	wav->wave.frequency = head.freq;
 	wav->wave.len = head.len;
 	wav->wave.loop_len = head.loop_len; 
-	wav->rom_addr = dfs_rom_addr(fn) + head.start_offset;
+	wav->currentFile = open(fn, O_RDONLY);
```

Comment:
I don't think there's a need to open a second file here, is there? You could just store the file descriptor here and avoiding closing it later.

Reply:
There are reads that happen later in this function, while the receiver of the wav struct assumes the file pointer not to have advanced from this position. I can do it the way you suggest but I'll have to lseek back because the file is not instanced.

Reply:
`head.start_offset` is not guaranteed to be the current position at that point; the format specifies the actual starting offset in the header specifically so that the body of data can begin at any point after the header. So you have to seek to `head.start_offset` before returning anyway.